backend/src/test-integration/java/com/crm/api/v1/CustomerControllerIT.java

```diff
@@ -178,6 +195,20 @@ void shouldResponseWith_Created_WhenAddedCustomer() throws Exception {
                 .andExpect(status().is(CREATED.value()));
     }
 
+    @Transactional
+    @Rollback
```

Comment:
I think @Transactional annotation on test has Rollback as default. @Rollback is not needed, but i'm not sure

Reply:
I will check that

Reply:
You might be right, thanks